components/navigation/navigation.css

```diff
@@ -16,15 +33,42 @@
   font-weight: 500;
   color: var(--black);
   text-decoration: none;
+  order: 2;
   transition: all 0.2s ease-in-out;
 }
-.nav .nav__links .nav__link {
+.nav__links.active {
+  display: flex;
+  visibility: visible;
+  opacity: 1;
+}
+@media (max-width: 768px) {
```

Comment:
I think it would be good if you can give at least a line space between two declarations. As it will increase the readability of the code. Like,

```css
.nav__links.active {
}

@media (max-width: 768px) {
}
```

Reply:
.css file is automatically generated from scss file. I have added spaces in scss file

Reply:
oh! okay. Great work.